src/Stratis.Bitcoin/Utilities/Extensions/ArgsExtensions.cs

```diff
@@ -1,29 +0,0 @@
-﻿using System.Linq;
-
-namespace Stratis.Bitcoin.Utilities.Extensions
-{
-    /// <summary>
-    /// Extension methods for arguments array.
-    /// </summary>
-    public static class ArgsExtensions
```

Comment:
Ah yes, this was much better, living in it's own extensions because it context specific to command line arguments.   Keeping this would be better and renaming to CommandLineArgsExtensions.